env-example

```diff
@@ -1,5 +1,24 @@
-MODULES=presto
-PRESTO_API=http://presto:8080/v1
+MODULES=presto,athena,prod,dev
+
+MODULES_PRESTO_API=http://presto:8080/v1
```

Comment:
since this will introduce multiple JDBC connections,
what about multiple presto/athena connections?